models/discordactions.js

```diff
@@ -47,16 +47,12 @@ const createNewRole = async (roleData) => {
 
 const removeMemberGroup = async (roleId, discordId) => {
   try {
-    const discordResponse = await removeRoleFromUser(roleId, discordId);
-    if (discordResponse) {
-      const backendResponse = await deleteRoleFromDatabase(roleId, discordId);
-      return backendResponse;
-    }
+    const backendResponse = await deleteRoleFromDatabase(roleId, discordId);
+    return backendResponse;
   } catch (error) {
     logger.error(`Error while removing role: ${error}`);
```

Comment:
Returning res.boom: It's unclear what res.boom does. If it's intended to send an error response, consider using standard Express methods like res.status(500).json({ message: 'Internal Server Error' }) for clarity.
Missing Error Details: The logged error message only includes ${err}, which might not provide sufficient information for debugging. Consider logging more specific details about the error, such as the error type, stack trace, and relevant variables.

Reply:
1. Did not understand the advice of not using res.boom. I have not added any code where I am returning res.boom.
2. Yes, you are right error logging should be more detailed. Creating a ticket for the same.
Ticket:https://github.com/Real-Dev-Squad/website-backend/issues/1861